Sources/FoundationEssentials/URL/URLTemplate.swift

```diff
@@ -176,6 +177,7 @@ extension URL.Template: CustomStringConvertible {
     }
 }
 
+@available(FoundationPreview 6.2, *)
```

Comment:
I don't think this annotation is needed since `URL.Template.Element` is an internal type, but otherwise LGTM!